app/controllers/assemblies_controller.rb

```diff
@@ -0,0 +1,29 @@
+class AssembliesController < ApplicationController
+  
+  def new
+    @assembly = Assembly.new
+  end
+
+  def create
+    @assembly = Assembly.new(assembly_params)
+    if@assembly.save
+       redirect_to new_part_path
+     else
+       render :new   
+    end  
+  end
+
+  def all_parts
+    @parts = []
+    p assembly =Assembly.where(id: params[:assembly_id]).first
+    #@parts = Assembly.where(id: params[:assembly_id]).first.try(:parts)
```

Comment:
@inquirlyaman Another best practice if the code is not needed. Remove before commit. Don't comment it.